docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+D/ Acknowledgement
+
+One has the freedom to precisely mention acknowledgements associated with the publication. 
+In practice, it is recommended to precise the grants or fellowships given to some authors.
+One should in any case always acknowledge the Astropy project, to which we are affiliated.
+
+Contributions in conferences
+-----------------------------
+
+The section concerns any contribution in conferences (a talk, a poster and their associated proceedings) related to
+the Gammapy project itself. It does not concern a technical or scientific work that uses Gammapy as an open library.
+In this case, the standard citation scheme should be used.
+
+As the length of the author list is generally a constrain, the author list can be reduced to the shortlist of authors,
+followed by 'The Gammapy Collaboration' associated with a link to the Gammapy team webpage.
+
+If there is a corresponding author, the 'Gammapy Coordination Committee', associated with its usual mailing list, is
+used. Concerning the acknowledgement, the Astropy project should always be mentioned.
+
+
+4. Storage in metadata files
+==================================
+
+Depending on the software repository, different metadata files are used in the eco-system of Open Source research
+software.
+
+The file ``codemeta.json`` is used by HAL and Software Heritage, and recommended by
+`ESCAPE <https://escape2020.pages.in2p3.fr/wp3/eossr/v0.3.2/metadata.html>`_ or
+`EOSC <https://www.eosc-pillar.eu/news/illustrating-benefits-eosc-research-communities-germany>`_.
+
+The file ``citation.cff`` is used by Zenodo and GitHub.
+
+5. Possible implementation
+==========================
+
+In order to respect these rules, some automation is required to create the list of contributors. One could use
+the Python library `Tributors <https://con.github.io/tributors/>`_. This library uses the GitHub API to determine the list of contributors, which is written in the
+format associated with codemeta.json, citation.cff, zenodo.json. This kind of library allows then two requirements:
+retrieve automatically the list of contributors from GitHub and write the authors list in all the needed formats.
+
+One should note that the user profile of the GitHub platform does not contain a field associated with the ORCID identifier.
+However, one could also retrieve the ORCID identifier of a contributor with the ORCID API
+(e.g. `How-To-1 <https://stackoverflow.com/questions/71052912/extracting-credentials-from-orcid-seach-with-orcid-id-using-python>`_
+or `How-To-2 <https://info.orcid.org/documentation/api-tutorials/api-tutorial-searching-the-orcid-registry/#easy-faq-2532>`_).
+
+Even if this library is chosen, a specific Python script located in the main Gammapy repository had to be written such
+that the authors' order follows these rules. In this case, integration within a GitHub action could be possible.
```

Comment:
Just as a side note, a fist version of such a script is herehttps://github.com/gammapy/gammapy/blob/master/dev/authors.py:

Reply:
This script is a starting point. But he 'Get list of authors from CITATION.cff', whereas here we wish to create the file author list! So, one should go beyond!